spec/presenters/supplier_number_presenter_spec.rb

```diff
@@ -19,5 +20,13 @@
         expect(presenter.supplier_label).to eq(account_number)
       end
     end
+
+    context 'when the supplier does not have a defined postcode' do
```

Comment:
I think this test should be described as `supplier does not have a defined name`